src/components/StarRating.tsx

```diff
@@ -20,6 +22,12 @@ const StarRating: React.FC<StarRatingProps> = ({
   const [starRating, setStarRating] = useState<number>(Math.round(voteAverage / 2));
   const [iconHover, setIconHover] = useState<number>(0);
 
+  const { theme }: ThemeContextType = useContext(ThemeContext);
+
+  if (!theme) {
+    colorUnfilled = '#ffffff';
```

Comment:
I guess it will be better to use variables here as "$white"

Reply:
to use as variables I need to  make "install sass-loader node-sass webpack", I guess it will update webpack config, right?

Reply:
no more relevant, as this logic moved to parent component